ci-operator/step-registry/openshift/agentic/trt/review-responder-eval/judge/openshift-agentic-trt-review-responder-eval-judge-commands.sh

```
@@ -0,0 +1,322 @@
+                fi
+            done < <(echo "${EXPECTED_FILES_JSON}" | jq -r '.[]')
+
+            if [[ "${ALL_EXPECTED_CHANGED}" == "true" ]]; then
+                record_check "${COMMENT_ID}_code_changed" "pass"
+            else
+                record_check "${COMMENT_ID}_code_changed" "fail"
+            fi
+
+            HAS_REPLY=false
+            if [[ -n "${GH_COMMENT_ID}" ]]; then
+                REPLY_COUNT=$(echo "${ALL_ISSUE_COMMENTS}" | jq --arg cid "${GH_COMMENT_ID}" '[.[] | select(.body | test("'"${COMMENT_ID}"'"; "i"))] | length' 2>/dev/null || echo "0")
```

Shellcheck (0.11.0)

[warning] REPLY_COUNT appears unused. Verify use (or export if used externally).

(SC2034)